src/cache_inode/cache_inode_misc.c

```diff
@@ -812,6 +814,7 @@ cache_inode_release_dirents(cache_entry_t *entry,
 			if (dirent->ckey.kv.len)
 				cache_inode_key_delete(&dirent->ckey);
 			gsh_free(dirent);
+      number_of_dirent_entries--;
```

Comment:
Fix indentation. Sent you an email on how your git should be configured. Also, run githooks script.

Reply:
I ran the script and fixed indentation.